.cirrus.yml

```
@@ -16,6 +16,9 @@ env:
   GRADLE_COMMON_FLAGS: --console plain --no-daemon
   GO_VERSION: 1.21.1
   PROTOC_VERSION: 25.0
+  ORG_GRADLE_PROJECT_signingKey: VAULT[development/kv/data/sign data.key]
```

```suggestion
  
  ORG_GRADLE_PROJECT_signingKey: VAULT[development/kv/data/sign data.key]
```
Let's make a clear separation